modules/serverless-sinkbinding-yaml.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// serverless/knstive_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-yaml_{context}"]
+= Using SinkBinding with the YAML method
+
+ This guide describes the steps required to create, manage and delete a SinkBinding instance using YAML files.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have the `kn` CLI installed.
+
+.Procedure
+
+. To check that SinkBinding is set up correctly, create a Knative event display service, or event sink, that dumps incoming messages to its log.
++
+Copy the following sample YAML into a file named `service.yaml`:
++
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-heartbeats:v0.13.2
```

Comment:
I'd use `quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2` here instead

Reply:
A little observation, the image should be the same in `kn` section vs `yaml` one. I'll add a comment to the exact place. 

The actual value from Lukas's comment is probably the best candidate.